app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -27,6 +28,9 @@ class PipelineSampleReport extends React.Component {
 
     this.state = {
       taxonomy_details: [],
+      backgroundName: Cookies.get('background_name') || this.report_details.background_model.name,
```

Comment:
New master doesn't have a background_model field in report_details anymore so you may need to refactor a bit